tornado/locks.py

```diff
@@ -430,6 +452,19 @@ class Lock(object):
     ...        pass
     ...
     ...    # Now the lock is released.
+
+    In Python 3.5, `Lock` also supports the async context manager protocol.
+    Note that in this case there is no `acquire`:
```

Comment:
The reason for this apparent inconsistency is a little subtle. I propose "Note that in this case there is no `yield lock.acquire()`, because the new `async with` statement provides a single place to wait for the lock *and* to enter a block of code."